### ISSUE TITLE
class with extends breaks WordPress/Sniff.php

### ISSUE THREAD

**Issue by themage** [WordPress/WordPress-Coding-Standards#1996]
## Bug Description
When Sniffing a file with a class without extends, WordPress/Sniff.php goes BOOM (like this):

```
FILE: /app/inc/email/class-justaclass.php
--------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------
 1 | ERROR | An error occurred during processing; checking has been aborted.
   |       | The error message was: Trying to access array offset on value of
   |       | type bool in
   |       | /app/vendor/wp-coding-standards/wpcs/WordPress/Sniff.php on line
   |       | 1252 (Internal.Exception)
--------------------------------------------------------------------------------
```
As this code, specifically, seems to be trying to find out if the class is a test class, I would expect this function to return false and move on.

Maybe a different rule should exist (or exists) to detect this if the code is not up to standards.

## Minimal Code Snippet

```php
<?php

class JustAClass {
}

```

## Error Code
See the description above.

## Environment

| Question               | Answer
| ------------------------| -------
| PHP version             | 8.0.7
| PHP_CodeSniffer version | 
| WPCS version            | 
| WPCS install type       | pistachio project docker-compose
| IDE (if relevant)       | 

## Tested Against `develop` branch?
- [ ] I have verified the issue still exists in the `develop` branch of WPCS.

**Comment by jrfnl** [comment]
Cannot reproduce the issue, not with WPCS 2.3.0, nor with WPCS 3.0.0 (`develop`).
Tested against PHPCS `develop`, `3.6.0` and even `3.5.0`.

Tested using PHP 8.0.6.

Note: PHPCS 3.6.0 is the first version of PHPCS which is (nearly) compatible with PHP 8.0.

```
class-justaclass.php
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES                                                                                                                                                                 
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 1 | ERROR | Missing file doc comment (Squiz.Commenting.FileComment.Missing)                                                                                                                     
 3 | ERROR | Missing doc comment for class JustAClass (Squiz.Commenting.ClassComment.Missing)                                                                                                    
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

The template is asking for versions for a reason.

**Comment by dingo-d** [comment]
@themage This used to happen on older PHPCS/WPCS versions If memory serves me (WPCS 2.2.1). You didn't mention what version of standards you are using.

**Comment by themage** [comment]
I'm sorry guys, this may be very old news.

If I'm reading this correctly, the project I'm looking at is asking for a version ^1.0.0 of wpcs, which was resolved to 1.2.1 - and phpcs is 3.1.0.

Should I close the issue and show myself out?

**Comment by GaryJones** [comment]
No problem - thanks for confirming the oldness!

I'll close this one out, and the presumption that you aren't able to recreate it with the latest versions. If you could open a PR for the upstream project that's still requiring ^1.0.0 of WPCS, then that will be appreciated to stop others tripping over the same issue.
